### PR TITLE
Add 3 decimal places to PossibleTimeSave

### PR DESCRIPTION
**Pull request by 10-days-till-xmas**

### CODE REVIEW

File: src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs
Comment (ero-qt) [inline src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs:1]: Please make sure the changes to the sizes and coordinates in this file are not caused by a display scale above or below 100%.

File: src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs [hunk 1]
```diff
@@ -213,145 +204,137 @@ private void InitializeComponent()
...
             // 
             this.btnTimeColor.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom) 
             | System.Windows.Forms.AnchorStyles.Left)));
             this.btnTimeColor.FlatStyle = System.Windows.Forms.FlatStyle.Popup;
-            this.btnTimeColor.Location = new System.Drawing.Point(224, 50);
-            this.btnTimeColor.Margin = new System.Windows.Forms.Padding(4, 5, 4, 5);
+            this.btnTimeColor.Location = new System.Drawing.Point(150, 32);
             this.btnTimeColor.Name = "btnTimeColor";
-            this.btnTimeColor.Size = new System.Drawing.Size(34, 35);
+            this.btnTimeColor.Size = new System.Drawing.Size(23, 23);
```
Comment (wooferzfg) [inline src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs:337]: Why are these changing?

File: src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs
Comment (wooferzfg) [reply src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs:1]: @10-days-till-xmas Please don't mark a comment as resolved unless it has been addressed

File: src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs [hunk 2]
```diff
@@ -451,23 +429,34 @@ private void InitializeComponent()
...
-            this.AutoScaleDimensions = new System.Drawing.SizeF(9F, 20F);
+            this.AutoScaleDimensions = new System.Drawing.SizeF(6F, 13F);
             this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
             this.Controls.Add(this.tableLayoutPanel1);
-            this.Margin = new System.Windows.Forms.Padding(4, 5, 4, 5);
             this.Name = "PossibleTimeSaveSettings";
-            this.Padding = new System.Windows.Forms.Padding(10, 11, 10, 11);
-            this.Size = new System.Drawing.Size(714, 569);
+            this.Padding = new System.Windows.Forms.Padding(7, 7, 7, 7);
+            this.Size = new System.Drawing.Size(476, 370);
```
Comment (wooferzfg) [inline src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs:459]: The size of the entire form changing is an indication that the scale was not set to 100%

Comment (10-days-till-xmas) [reply src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs:459]: I was using 100% scaling the whole time though? Idk I'll fix it anyways

Comment (wooferzfg) [reply src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs:459]: It seems like the last person who changed this file may not have been using 100% scaling. This PR is fine then

**Pull request merged**